domain/src/main/java/org/oppia/domain/profile/ProfileTestHelper.kt

```diff
@@ -51,4 +63,37 @@ class ProfileTestHelper @Inject constructor(
   /** Login to user profile. */
   fun loginToUser() =
     profileManagementController.loginToProfile(ProfileId.newBuilder().setInternalId(1).build())
+  /** Login to user profile. */
+  fun loginToUser2() =
+    profileManagementController.loginToProfile(ProfileId.newBuilder().setInternalId(2).build())
+
+  fun getStoryTextSize(storyTextSize: StoryTextSize): Any {
+    return when (storyTextSize) {
+      StoryTextSize.SMALL_TEXT_SIZE -> 16f
```

Comment:
@veena14cs I am still not sure if `16f` should be here or not. I will need to defer to @BenHenning for this.

Reply:
I have removed this function and changed testcases accordingly please check.